src/components/OrganizationFilter.js

```diff
@@ -95,55 +87,185 @@ export const useOrganizationFilterState = () => {
+  currentFilter,
+  categories,
+  pageContext,
+  onApplyFilter,
+  organizations,
+}) => {
+  const { byCategory, byHeadcount, byOrgType } = currentFilter
+
+  const formattedSubcategories = pageContext
+    ? formatSubcategoryOptions(categories, pageContext)
+    : null
+
+  const formattedHeadCounts = pageContext
+    ? formatHeadcounts(organizations)
+    : null
+  const formattedOrgTypes = pageContext ? formatOrgTypes(organizations) : null
+
+  const styles = {
+    container: styles => ({
+      ...styles,
+      display: "inline-block",
+      marginRight: "8px",
+      maxWidth: "30%",
+      maxHeight: "24px",
+      verticalAlign: "middle",
+    }),
+    control: provided => ({
+      ...provided,
+      minHeight: "24px",
+      maxHeight: "24px",
+      borderRadius: "9999px",
+      maxWidth: "100%",
+      backgroundColor: "#f7fafc",
+    }),
+    indicatorsContainer: provided => ({
+      ...provided,
+      height: "24px",
+    }),
+    indicatorSeparator: provided => ({
+      ...provided,
+      display: "none",
+    }),
+    clearIndicator: provided => ({
+      ...provided,
+      padding: "5px",
+    }),
+    dropdownIndicator: provided => ({
+      ...provided,
+      padding: "0px",
+      paddingRight: "5px",
+    }),
+    placeholder: provided => ({
+      ...provided,
+      position: "static",
+      top: "auto",
+      transform: "none",
+      maxWidth: "100%",
+      overflow: "hidden",
+      textOverflow: "ellipsis",
+      whiteSpace: "nowrap",
+    }),
+    menu: provided => ({
+      ...provided,
+      width: "max-content",
+      minWidth: "100%",
+    }),
+
+    option: (provided, { isFocused }) => {
+      return {
+        ...provided,
+        backgroundColor: isFocused ? "#e2e8f0" : null,
+      }
+    },
+    singleValue: provided => ({
+      ...provided,
+      maxWidth: "none",
+      position: "static",
+      top: "auto",
+      transform: "none",
+    }),
+    valueContainer: provided => ({
+      ...provided,
+      overflow: "hidden",
+      textOverflow: "ellipsis",
+      maxWidth: "100%",
+    }),
+  }
 
   return (
     <>
-      {hasFilterApplied && (
-        <p className="text-gray-700  text-sm">
-          <span className="mr-2">Filtered by</span>
-          {byCategory && <OrganizationCategory active text={byCategory.name} />}
-          {byLocation && <OrganizationLocation active text={byLocation} />}
-          {byHeadcount && <OrganizationHeadcount active text={byHeadcount} />}
-          {byOrgType && <OrganizationOrgType active text={byOrgType} />}
-          {byCapitalType && (
-            <OrganizationCapitalType active text={byCapitalType} />
-          )}
-          {byCapitalStrategic && <OrganizationCapitalStrategic active />}
-          {byCapitalStage && (
-            <OrganizationCapitalStage active text={byCapitalStage} />
-          )}
-          {byCapitalCheckSize && (
-            <OrganizationCapitalCheckSize active text={byCapitalCheckSize} />
-          )}
```

Comment:
Can check size should also be dropdowns on the capital page.

I'm not sure what to do with strategic which is a bool

Reply:
Makes sense. I will work on the capital page.